spec/imsc-hrm.html

```diff
@@ -728,7 +728,26 @@ <h2>Acknowledgements</h2>
 
   <section class="appendix informative">
     <h2>Privacy and Security Considerations</h2>
-
+    <p>This specification has no inherent security or privacy implications.</p>
+    <p>The algorithm defined within this specification is used for static analysis of a resource.
+      This specification does not define any protocol or interface for obtaining such a resource,
+      and it does not define any interface for exposing the results of the analysis.
+      No personal or sensitive information is processed as part of the algorithm,
+      other than any such information that might happen to be part of the IMSC document being analysed.
+      No information is exposed by the algorithm to any origin.
+      No scripts are loaded or processed as part of the algorithm
+      and no links to external resources are dereferenced.
+    </p>
+    <section>
```

Comment:
```suggestion
     </section>
    <section>
```

Avoiding hanging paragraphs